src/gateways/BaseGoPay.php

```diff
@@ -388,7 +388,7 @@ protected function prepareDescription($paymentItems, IRow $payment): string
         return $this->applicationConfig->get('site_title');
     }
 
-    protected function isPendingState(string $state): bool
+    protected function isPendingState(?string $state = null): bool
```

Comment:
In an offline conversation you told me that the `state` parameter might be completely missing. I've checked the usages and I found one place that probably needs to be handled to avoid notices:

https://github.com/remp2020/crm-gopay-module/blob/0171bbdc117ae4b53dee729ebf86075079b3d539/src/gateways/BaseGoPay.php#L134

Can you please check (and possibly test) it? The remaining places look OK to me, I'd merge it afterwards.